viewer/packages/cad-geometry-loaders/src/sector/SectorLoader.ts

```diff
@@ -93,6 +93,7 @@ export class SectorLoader {
     const currentBatchId = this._batchId;
 
     for (const batch of chunk(changedSectors, SectorLoadingBatchSize)) {
+      if (currentBatchId !== this._batchId) break; // Stop processing this batch as a new batch has started, and will discard results from old batches.
```

Comment:
Line 101 discards any result, so this cancels out early before sending the request at all.